packages/react/src/FAQ/FAQ.stories.tsx

```diff
@@ -30,35 +30,19 @@ export const Default = () => (
       <FAQ.Answer>
         <p>Startups who meet the following criteria are eligible to apply for the program:</p>
         <ol>
-          <li>
-            <Text size="300" variant="muted">
-              Must be associated with a current GitHub for Startups partner.
-            </Text>
-          </li>
-          <li>
-            <Text size="300" variant="muted">
-              Self-funded or funded (Seed-Series A)
-            </Text>
-          </li>
-          <li>
-            <Text size="300" variant="muted">
-              Not a current GitHub Enterprise customer
-            </Text>
-          </li>
-          <li>
-            <Text size="300" variant="muted">
-              Must not have previously received credits for GitHub Enterprise
-            </Text>
-          </li>
+          <li>Must be associated with a current GitHub for Startups partner.</li>
+          <li>Self-funded or funded (Seed-Series A)</li>
+          <li>Not a current GitHub Enterprise customer</li>
+          <li>Must not have previously received credits for GitHub Enterprise</li>
```

Comment:
Same question here. I did notice the list style is different because it is bulleted. I think this might be something that we could request be changed via design as well. What do you think?

Reply:
> I did notice the list style is different because it is bulleted.

Thanks, I'll review the diffs to see what's going on. I haven't changed the <ol> though so wonder why it's showing bullets 🤔

The main change here is removing the unnecessary Text overrides.

Reply:
Yes, that is fair it doesn't need to be part of this PR. I was just noticing an update we could make seeing that you made a similar one earlier. But this is fine without this change too.